ninja_syntax.py

```diff
@@ -19,18 +19,18 @@ def __init__(self, output, width=78):
         self.width = width
 
     def newline(self):
-        self.output.write('\n')
+        self.output.write('\n'.encode('utf-8'))
```

Comment:
I've used this file in the past with py3 unmodified. Are you sure you need to make these changes? If so, maybe try to get them upstreamed to https://github.com/ninja-build/ninja/commits/master/misc/ninja_syntax.py, since it is supposed to be py2/3 agnostic.

Reply:
It depends entirely on what output is. I guess I should look where we're creating this and make sure we aren't passing a stream which expects bytes.